src/lib/pickles/pickles.ml

```diff
@@ -1154,6 +1154,8 @@ module Make_str (_ : Wire_types.Concrete) = struct
 
       let () = Tick.Keypair.set_urs_info []
 
+      let () = Backtrace.elide := false
```

Comment:
perhaps worthy of a comment in the code? If I read this line I'm not sure what it is meant to do